_docs/graphql-api.md

```diff
@@ -456,15 +456,16 @@ query event_by_tx {
 ```graphql
 {
   accountSnapshots(
-    first: 100,
+    first: 10,
```

Comment:
Could querying less make the query faster? For the first try users don’t usually need 100 results, so I suggest to give them less not to overwhelm them.